README.md

```diff
@@ -1,21 +1,36 @@
 # DevOps Apprenticeship: Project Exercise
 
-## Getting started
+## System Requirements
 
-The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal:
+The project uses poetry for Python to create an isolated environment and manage package dependencies. To prepare your system, ensure you have an official distribution of Python version 3.7+ and install poetry using one of the following commands (as instructed by the [poetry documentation](https://python-poetry.org/docs/#system-requirements)):
+
+### Poetry installation (Bash)
 
-### On macOS and Linux
 ```bash
-$ source setup.sh
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
 ```
-### On Windows (Using Git Bash)
+
+## Dependencies
+
+The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from your preferred shell:
+
 ```bash
-$ source setup.sh --windows
+$ poetry install
 ```
 
-Once the setup script has completed and all packages have been installed, start the Flask app by running:
+You'll also need to clone a new `.env` file from the `.env.template` to store local configuration options. This is a one-time operation on first setup:
+
 ```bash
-$ flask run
+$ cp .env.template .env  # (first time only)
+```
+
+The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
```

Comment:
This should be updated to mention the env vars you actually require rather than the unused SECRET_KEY